src/components/Navigation.js

```diff
@@ -41,6 +41,7 @@ function Navigation() {
   const [showSearchInput, setShowSearchInput] = useState(false);
   const [showSidebar, setShowSidebar] = useState(false);
   const showTokensTab = isTokensBalanceEnabled || isTokensBaseEnabled;
+  const network = hathorLib.config.getNetwork();
```

Comment:
I'm sorry I wasn't more clear on the last review.
I forgot the explorer is built with the `REACT_APP_NETWORK` constant that holds the network.

We can just import `REACT_APP_NETWORK` from '../constants', it will have `mainnet` or `testnet` we want to use. ([reference](https://github.com/julia-eileen/hathor-explorer/blob/feat/revamp-home-ui/src/constants.js#L12)).